src/OpenLoco/src/Ui/Windows/StationWindow.cpp

```diff
@@ -1050,6 +1534,85 @@ namespace OpenLoco::Ui::Windows::Station
                     }
                 }
             }
+
+            // clang-format off
+            static std::pair<WidgetIndex_t, std::array<uint32_t, 8>> tabAnimations[] = {
```

Comment:
should this be const? and can it be constexpr and kTabAnimations. (Althought i guess this has been copied)

Reply:
Indeed, nice catch. I've applied the same change in the original spot.